daemon/mgr/container.go

```diff
@@ -706,8 +706,9 @@ func (mgr *ContainerManager) Stop(ctx context.Context, name string, timeout int6
 	c.Lock()
 	defer c.Unlock()
 
-	if !c.IsRunning() {
-		return fmt.Errorf("container's status is not running: %s", c.meta.State.Status)
+	if c.IsStopped() {
```

Comment:
create a new container, it's status is "created", actually, the container is stopped, but `IsStopped()` will return false